app/adapters/application.coffee

```diff
@@ -0,0 +1,11 @@
+`import DS from 'ember-data'`
+
+ApplicationAdapter = DS.ActiveModelAdapter.extend 
+  host: 'http://localhost:3000/api/v1',
```

Comment:
Rather than hard coding the host for development, the better practice (I've just learned) is to just use relative paths and, when serving up the ember files in development, use the --proxy option to proxy local app requests to a rails backend. For production, the setup will likely be similar: apache serving up the ember files and proxying api requests to the rails backend.